src/react-apps/ux-editor/src/reducers/formDesignerReducer/formLayoutReducer.ts

```diff
@@ -199,6 +199,29 @@ const formLayoutReducer: Reducer<IFormLayoutState> = (
         },
       });
     }
+    case FormDesignerActionTypes.UPDATE_FORM_CONTAINER_FULFILLED: {
+      const { updatedContainer, id } = action as FormDesignerActions.IUpdateFormContainerActionFulfilled;
+      return update<IFormLayoutState>(state, {
+        containers: {
+          [id]: {
+            $apply: () => ({ ...updatedContainer }),
```

Comment:
Is this `updatedContainer` a nested object? Because the spread-operator `...` only makes a new object shallowly. This might be the cause of our wierd side-effects somehow.

Reply:
The updatedContainer is not a nested object, so this should not be the cause of our problems. 
I see that the spread-operator is used in UPDATE_FORM_COMPONENT_FULFILLED, where the updatedComponent is a nested object (?). Could this be relevant? Maybe @nkylstad  knows more?

Reply:
updatedComponent is not a nested object, its similar to updatedContainer 
However maybe we should look into trying to change both of these to not use the spread-operator to see if this fixes the wierd side-effects we are having

Reply:
Yeah, keep it as it is, and we will do a cleanup after investigating.